src/components/Account/Menu.tsx

```diff
@@ -35,9 +48,12 @@ const AccountMenu: React.FC<BoxProps> = (props) => {
         />
         <MenuList>
           <MenuItem as={RouterLink} to={Routes.dashboard.profile}>
+            <ImProfile size={16} />
             <Trans i18nKey='profile'>Profile</Trans>
           </MenuItem>
+          <MenuDivider />
           <MenuItem onClick={logout}>
+            <TbLogout size={20} />
```

Comment:
Avoid using icons directly. Use them via the `Icon` chakra-ui component, which allows greater control over them.